quasar-finance/quasar
# Revision of variables and workflows

## 1. Overview

This PR focuses on refactoring the existing CosmWasm contract to improve code readability and maintainability. The primary changes involve renaming variables for consistency, reorganizing the file structure, and separating functions into different files.

Co-author: @nseguias @LaurensKubat 

## 2. Implementation details

- Variable Renaming: All variables have been renamed to follow a more consistent naming convention. This enhances code readability and ensures that future developers can easily understand the purpose of each variable.
- Function Separation: Functions that were previously part of large files have been moved to their own separate files. This modular approach makes it easier to manage and locate specific functionalities within the codebase.
- Reorganized File Structure: The existing file structure has been reorganized to follow a more logical hierarchy, making it easier to navigate the project.
- Added new state CURRENT_SWAP
- Logical changes on range.rs implementing CURRENT_SWAP load
- Minor logical changes on deposit.rs, withdraw.rs

## 3. How to test/use

1. Run the usual build and test procedures to ensure that the contract behaves as expected. No logical changes have been made, so existing test cases should all pass.
2. Navigate the updated file structure to ensure that it is easier to find relevant sections of code and that the new naming conventions make sense.

## 4. Checklist



- [ ] Does the Readme need to be updated?

## 5. Limitations (optional)

There are no new limitations introduced by this PR. All capabilities listed previously remain unchanged.

## 6. Future Work (optional)

Future work could involve further modularization of the code, possibly dividing it into separate packages or libraries to improve maintainability and reusability.
